### ISSUE TITLE
Does not work with TailwindCSS anymore

### ISSUE THREAD

**Issue by fmvilas** [react-syntax-highlighter/react-syntax-highlighter#311]
**Is your feature request related to a problem? Please describe.**
We're using TailwindCSS to style our website. Just upgraded this component for security reasons and found that `lineNumberContainerProps` and `lineNumberProps` were removed. Also, `lineProps` is not allowing us to pass custom classNames. That makes it impossible for us to use both Tailwind and this component.

You can see more here: https://github.com/asyncapi/website/blob/master/components/editor/CodeBlock.js#L61-L77.

**Describe the solution you'd like**
For `lineNumberProps` and `lineNumberContainerProps` either bring them back or maybe create something like `lineNumberClassNames` and `lineNumberContainerClassNames`. Happy to discuss what's better.

For `lineProps`, it seems the offending line is [this one](https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/master/src/highlight.js#L109). Maybe this can be changed to:

```js
properties.className = properties.className ? `${className} ${properties.className}` : className;
```

**Describe alternatives you've considered**
I've considered requiring the Tailwind configuration file from the component but that would couple the component to the config file, making it less reusable. Also considered hard-coding the values I want as inline styles but we'd get duplicated values in the config file and in this component.

---

Happy to help implementing the solution to this. Thanks for the great work! 🙌

**Comment by simmerer** [comment]
Good catch! Looks like there was a PR to get rid of `lineNumberContainerStyle` and `lineNumberStyle` and replace them with   `lineNumberContainerProps` and `lineNumberProps`: https://github.com/react-syntax-highlighter/react-syntax-highlighter/pull/229 ... but somehow it was regressed with the 13.0.0 update.

If you've got a bit of time to implement a solution, we'd be grateful.

**Comment by fmvilas** [comment]
Absolutely. Do you want me to cherry pick the changes on #229?

**Comment by simmerer** [comment]
Go for it!

**Comment by SalahAdDin** [comment]
@simmerer I'm having [problems](https://github.com/react-syntax-highlighter/react-syntax-highlighter/pull/320#issuecomment-743785602) with it too.
